jupyter_telemetry/eventlog.py

```diff
@@ -44,15 +30,12 @@ class EventLog(Configurable):
     """
     Send structured events to a logging sink
     """
-    # Traitlets can't take non-pickleable objects, so can't pass in
-    # logging handlers directly. This is a temporary hack we should fix?
-    handlers_maker = Callable(
+    handlers_list = HandlersList(
         None,
         config=True,
         allow_none=True,
-        help="""
```

Comment:
Not 100% familiar with Traitlets, but what is the use-case for allowing "None" handlers? Where would we publish the events to?

Reply:
They would go nowhere, which is the default. They'd get dropped when received.

Reply:
Yea my poiint I don't get the user-case for creating a EventLog with a None handler since its basically useless and might be potentially confusing to the user.